datamade/la-metro-councilmatic
# Rebase log-sources

## Overview

Rebases log-sources branch.
This is based off PR #935

Closes #910 

## Testing Instructions

 * Navigate to the detail page for a person, event, or bill
 * Open the console to check if it's source has been printed
 * Follow link(s) present
